pulseapi/settings.py

```diff
@@ -201,6 +201,7 @@ def show_toolbar(request):
 # and <version value> is the value of the version that will be used in URLs
 API_VERSION_LIST = [
     ('version_1', 'v1',),
+    ('version_2', 'v2',),
 ]
 DEFAULT_VERSION = 'version_1'
```

Comment:
I presume we'll default to v2 when the front end catches up?

Reply:
I don't have a pref, up to you and @alanmoo

Reply:
Once all our properties are using versioning in the routes they call, I'd say we can update the default to always reflect the latest version.